Helper/Ratings.php

```diff
@@ -233,6 +237,20 @@ class_exists('Yotpo\Yotpo\Helper\RichSnippets') &&
         return false;
     }
 
+    /**
+     * Check if the Review module is enabled, review tables are present
+     *
+     * @return bool
+     */
+    public function canUseMagentoRatingsAndReviews()
+    {
+        if ($this->moduleManager->isEnabled('Magento_Review')) {
```

Comment:
You can return this function directly, no need for an if statement

Reply:
👍